test/e2e/serial/tests/workload_overhead.go

```diff
@@ -225,8 +230,7 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload overhea
 				Expect(err).NotTo(HaveOccurred(), "Unable to get pods from Deployment %q:  %v", deployment.Name, err)
 
 				podResourcesWithOverhead := podResources.DeepCopy()
-				podResourcesWithOverhead.Cpu().Add(*rtClass.Overhead.PodFixed.Cpu())
-				podResourcesWithOverhead.Memory().Add(*rtClass.Overhead.PodFixed.Memory())
+				resourcelist.AddCoreResources(podResourcesWithOverhead, *rtClass.Overhead.PodFixed.Cpu(), *rtClass.Overhead.PodFixed.Memory())
```

Comment:
note that the pod should be guaranteed, yet in line 213 it sets only the limits

Reply:
this is fine: https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed
```
Note: If a Container specifies its own memory limit, but does not specify a memory request, Kubernetes automatically assigns a memory request that matches the limit. Similarly, if a Container specifies its own CPU limit, but does not specify a CPU request, Kubernetes automatically assigns a CPU request that matches the limit. 
```

Reply:
good to know that, thanks for the link!